### PR TITLE
Add set and get function

### PR DESCRIPTION
**Pull request by Bhavin9898**
I have added set and get function to remove error for new user.

I am targeting this branch, because I have face problem without this code.

## Changelog
```markdown
### Added
- Added get and set method.
```

## Subject

I have added get and set method in entity file.

### CODE REVIEW

File: docs/getting_started/creating_an_admin.rst [hunk 1]
```diff
@@ -61,6 +61,12 @@ two very simple ``Post`` and ``Tag`` entities will be used::
     class Category
     {
         // ...
+       /**
+         * @ORM\Id
+         * @ORM\Column(type="integer")
+         * @ORM\GeneratedValue(strategy="AUTO")
+         */
+        protected $id;
```
Comment (core23) [inline docs/getting_started/creating_an_admin.rst]: Why `protected`?

File: docs/getting_started/creating_an_admin.rst [hunk 2]
```diff
@@ -83,7 +89,36 @@ two very simple ``Post`` and ``Tag`` entities will be used::
...
+        /**
+         * Set name
+         *
+         * @param string $name
+         * @return Category
+         */
+        public function setName($name) {
+            $this->name = $name;
+
+            return $this;
```
Comment (core23) [inline docs/getting_started/creating_an_admin.rst]: Why?

File: docs/getting_started/creating_an_admin.rst (hunk 1 above)
Comment (kunicmarko20) [reply docs/getting_started/creating_an_admin.rst]: Isn't the point of `// ...` data people should already have in their entity or in this case this `$id`.

**Comment by Bhavin9898** [comment]
Some new user can forget to add set and get function to file.

File: docs/getting_started/creating_an_admin.rst (hunk 2 above)
Comment (Bhavin9898) [reply docs/getting_started/creating_an_admin.rst]: Hii Core23 I have changed protected to private.

File: docs/getting_started/creating_an_admin.rst (hunk 1 above)
Comment (kunicmarko20) [reply docs/getting_started/creating_an_admin.rst]: But that would mean that the new user didn't even bother to read Symfony/Doctrine docs, I don' t think we should add this, cc @sonata-project/contributors?

Comment (OskarStark) [reply docs/getting_started/creating_an_admin.rst]: I am against this change

File: docs/getting_started/creating_an_admin.rst [hunk 3]
```diff
@@ -83,7 +89,36 @@ two very simple ``Post`` and ``Tag`` entities will be used::
         {
             return $this->blogPosts;
         }
+        
+        /**
+         * Get id
+         *
+         * @return integer 
+         */
+        public function getId() {
```
Comment (OskarStark) [inline docs/getting_started/creating_an_admin.rst]: please put the `{` brackets on the next line

Comment (Bhavin9898) [reply docs/getting_started/creating_an_admin.rst]: @OskarStark Thank you sir.
I have done this thing

**Comment by kunicmarko20** [comment]
I don't see a point of this PR we on purpose leave `//...` so our docs aren't too big. If you do not know you need getters/setter or an id then you should read about PHP property visibility or how to use doctrine.

**Comment by Bhavin9898** [comment]
@kunicmarko20  You are right sir. I think this will make long document.

**Pull request closed without merging**